cmd/controller-bootstrap/command/apiutil_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package command
+
+import (
+	"go/build"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	goPath               = build.Default.GOPATH
+	defaultRootDirectory = filepath.Join(goPath, "src/github.com/aws-controllers-k8s")
```

Comment:
You don't need this. You can just find the files relative to your working directory.
Look at this for example: https://github.com/aws-controllers-k8s/code-generator/blob/main/pkg/testutil/schema_helper.go#L59

This is because sometimes the package may not be cloned in GOPATH.